handler/commentHandler_test.go

```diff
@@ -515,3 +515,44 @@ func Test_isDcoLabel(t *testing.T) {
 		})
 	}
 }
+
+func Test_Parsing_Reviewers(t *testing.T) {
+
+	var reviewersOptions = []struct {
+		title        string
+		body         string
+		expectedType string
+		expectedVal  string
+	}{
+		{
+			title:        "Right set reviewer",
```

Comment:
What does `Right set reviewer` mean? This is a positive test case I think? Maybe there is another more idiomatic way of describing the test.. @rgee0 any ideas?

Reply:
`Correct set reviewer`
`Set Reviewer correctly`
`set reviewer as expected`
`set reviewer as intended`

Reply:
The test scenario is testing a function named `parse`. What about `Parse request to assign reviewer` or `Parse the reviewer given a valid username`?